### ISSUE TITLE
Bug: nginx logs geopip not resolving

### ISSUE THREAD

**Issue by heyhewmike** [bobbyiliev/quick_access_logs_summary#3]
### Describe the bug

When running the script I am getting back alot of http code instead of the IP and Geolocation for the IP when attempting to perform the geo location check for the IPs discovered by the scripts in the logs

### To Reproduce

Running on Debian 10
Installed nginx via apt
Using default logging
chmod +x to the script

ran the script with the proper /path/to/nginx/access.log

### Expected behavior

Provide geolocation of the IP address

### Screenshots

See attached error file

### Additional context

[spike_geopip_error.txt](https://github.com/bobbyiliev/quick_access_logs_summary/files/8364349/spike_geopip_error.txt)

**Comment by bobbyiliev** [comment]
Thanks for reporting this! I will look into it further today!

**Comment by bobbyiliev** [comment]
Just tested this but it seems to be working as expected. Did you by any chance change the `geoipdomain` variable? It should be set to `geoipdomain="https://bobbyiliev.com/ip.php"`

**Comment by heyhewmike** [comment]
At the time I ran the script to get the error I had made no changes to your script.
I am running nginx/1.14.2 instead of apache.
I have set the geo variable and commented out the read for geo so that I can run the script as part of a cronjob and not have it hang at the prompt for input

![image](https://user-images.githubusercontent.com/75391794/160828612-40c37ebe-89a3-42b4-ab36-7ec1b8a015d6.png)

**Comment by bobbyiliev** [comment]
Just had another look at it looks like that it has been Cloudflare blocking some of your requests, are you still seeing the issue? I might change how the geo check is done so that it does not go over Cloudflare

**Comment by heyhewmike** [comment]
Yes. Now that you mention Cloudflare I do see that DDoS protection by html code in that mess of code I got when I just ran it again.

**Comment by bobbyiliev** [comment]
Just created a new endpoint, can you try changing the `geoipdomain` to:

```
https://cf-geoip.bobbyiliev.workers.dev/
```

And then give it another try?

**Comment by heyhewmike** [comment]
I am still getting a CloudFlare Captcha page being returned. I am also seeing text being returned instructing me to turn on Javascript and reload the page. I am also being instructed to enable Cookies and reload.

**Comment by bobbyiliev** [comment]
Hi there @heyhewmike,

Can you try with the following:

```
geoipdomain="https://geoip-cb4ki.ondigitalocean.app/default/geoip"
```

This is a new serverless function deployed on DigitalOcean

**Comment by heyhewmike** [comment]
Hi There @bobbyiliev,

Thank you. That worked. Interesting to see what countries have pinged the site.
